### DISCUSSION TITLE
[Support] midway的源码中引用依赖报错，如何解决？

### DISCUSSION THREAD

**Discussion by phoooob** [midwayjs/midway#3626] [Q&A] [answered]
### Describe the problem（描述问题）

安装了所有依赖了，但是这行依然会报错，感觉应该是要指向 /packages/core 模块，应该如何处理？

![1707041929976](https://github.com/midwayjs/midway/assets/13043634/ce0e13e6-50d6-437e-b78e-82fe3d03e671)

### Midway Versions（Midway 版本）

npx midway-version@latest

**Comment by czy88840616** [comment, marked as answer]
npm i& npm run build
